extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -711,8 +713,8 @@ AMP.extension(TAG, '0.1', AMP => {
   AMP.registerServiceForDoc(
       'amp-analytics-instrumentation', InstrumentationService);
   AMP.registerServiceForDoc('activity', Activity);
-  installVariableService(AMP.win);
   installLinkerReaderService(AMP.win);
+  AMP.registerServiceForDoc('amp-analytics-variables', VariableService);
```

Comment:
This is where the previous PR was broken. It was written as `installVariableServiceForDoc(AMP.ampdoc);` and `AMP.ampdoc` was not doing the right thing in the shadow case.

Reply:
Thanks for the heads up 😄

Reply:
so should we remove `installVariableServiceForDoc` if it's dangerous?

Reply:
I kind of did? It is now `installVariableServiceForTesting` and only used in tests.